src/index.scss

```diff
@@ -1,19 +1,22 @@
-@import "variables.css";
+@import '~hds-core/lib/base.css';
+@import 'font';
+@import 'variables';
+
 
 html {
   height: 100%;
   margin:0;
 }
 
 body {
-  margin: 0;
+  color: var(--color-black-90);
```

Comment:
We discussed that these hds variables could be abstracted to hauki specific variables, and then in our code only apply to hauki specific variables